packages/menu/test/menu.test.ts

```diff
@@ -707,4 +707,249 @@ describe('Menu', () => {
         await nextFrame();
         expect(el.selected).to.deep.equal(['3']);
     });
+
+    it('prevents selection when scrolling is detected', async () => {
+        const el = await fixture<Menu>(html`
+            <sp-menu selects="single">
+                <sp-menu-item value="1">Item 1</sp-menu-item>
+                <sp-menu-item value="2">Item 2</sp-menu-item>
+            </sp-menu>
+        `);
+        await elementUpdated(el);
+
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const menu = el as any;
+        const firstItem = el.querySelector('sp-menu-item') as MenuItem;
+
+        // Manually set scrolling state to simulate iPad scrolling
+        menu.isScrolling = true;
+
+        // Try to select an item while scrolling
+        const clickEvent = new MouseEvent('click', { button: 0 });
+        firstItem.dispatchEvent(clickEvent);
+        await elementUpdated(el);
+        await elementUpdated(firstItem);
+
+        // Verify selection is prevented
+        expect(firstItem.selected).to.be.false;
+        expect(el.value).to.equal('');
+    });
+
+    it('allows selection when not scrolling', async () => {
+        const el = await fixture<Menu>(html`
+            <sp-menu selects="single">
+                <sp-menu-item value="1">Item 1</sp-menu-item>
+                <sp-menu-item value="2">Item 2</sp-menu-item>
+            </sp-menu>
+        `);
+        await elementUpdated(el);
+
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const menu = el as any;
+        const firstItem = el.querySelector('sp-menu-item') as MenuItem;
+
+        // Verify not scrolling initially
+        expect(menu.isScrolling).to.be.false;
+
+        // Try to select an item when not scrolling
+        firstItem.click();
+
+        // Wait for the selection to be processed
+        await elementUpdated(el);
+        await elementUpdated(firstItem);
+        await nextFrame();
```

Comment:
this shouldnt be needed after an elementUpdated returns. Can you explain why this is needed?